ci_framework/roles/set_openstack_containers/tasks/main.yml

```diff
@@ -107,7 +107,11 @@
 
 - name: Get the operator pod name
   ansible.builtin.set_fact:
-    operator_pod_name: "{{ '\n'.join(operator_pod_name.stdout_lines) |regex_search('^pod/' ~ operator_name ~ '-operator-controller-manager-.*$', multiline=True) }}"
+    operator_pod_name: >-
+      {{
+      '\n'.join(operator_pod_name.stdout_lines) |
+      regex_search('^pod/' ~ operator_name ~ '-operator-controller-manager-.*$', multiline=True)
+      }}
```

Comment:
we need to change this to 
```
ansible.builtin.set_fact:
         operator_pod_name: 
            "{{
            '\n'.join(operator_pod_name.stdout_lines) |
            regex_search('^pod/' ~ operator_name ~ '-operator-controller-manager-.*$', multiline=True)
            }}"
```
it will work.